### ISSUE TITLE
Can you support the OpenHarmony JavaScript version?

### ISSUE THREAD

**Issue by DoraCoder** [albfernandez/juniversalchardet#47]
I am developing an OpenHarmony application that uses the JavaScript language. Can you support the OpenHarmony JavaScript version? I am willing to participate in the development and look forward to reply.

**Comment by albfernandez** [comment]
Hi

I've only maintaining this abandoned library, I've not time or knowledge to migrate to another language, sorry.
But maybe someone else can help.

**Comment by amake** [comment]
I don't think it makes a lot of sense to add JavaScript support to this library. A separately library of similar provenance for JavaScript would be [jschardet](https://github.com/aadsm/jschardet).
